awscodecommit/Dockerfile

```diff
@@ -1,25 +1,31 @@
-FROM golang:alpine AS builder
+FROM golang:1.13-alpine AS builder
+
+ENV GO111MODULE on
+ENV CGO_ENABLED 0
+ENV GOOS linux
+ENV GOARCH amd64
 
-COPY awscodecommit /go/src/awscodecommit
 WORKDIR /go/src/awscodecommit
 
 RUN apk update && apk add --no-cache openssh-client ca-certificates
-RUN GO111MODULE=on go mod download && \
-    CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -installsuffix cgo -o /go/bin/awscodecommit
 
-FROM registry.access.redhat.com/ubi8/ubi-minimal
+COPY awscodecommit/go.mod awscodecommit/go.sum /go/src/awscodecommit/
+RUN go mod download
 
-LABEL name="Triggermesh AWS CodeCommit Event Source" \
-      vendor="Triggermesh" \
-      version="v0.1.0" \
-      release="1" \
-      summary="The Triggermesh CodeCommit Source" \
-      description="This is the Triggermesh Knative Event Source for AWS CodeCommit"
+COPY awscodecommit/ /go/src/awscodecommit/
+RUN go install -installsuffix cgo ./...
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal
 
-RUN microdnf update
-COPY ./licenses/* /licenses/
+LABEL name "Triggermesh AWS CodeCommit Event Source"
+LABEL vendor "Triggermesh"
+LABEL version "v0.1.0"
+LABEL release "1"
+LABEL summary "The Triggermesh CodeCommit Source"
+LABEL description "This is the Triggermesh Knative Event Source for AWS CodeCommit"
```

Comment:
Same comment as above, regarding the grouping of `ENV`.

Reply:
one quick comment about those labels so that everyone knows. They are mandatory to pass the RedHat certification scanning. You can edit them but don't delete them

Reply:
@antoineco  same response as above :wink: 
@sebgoa thanks for the clarification